rootfs/api/urls.py

```diff
@@ -106,7 +106,7 @@
     url(r'^auth/tokens/$',
         views.TokenManagementViewSet.as_view({'post': 'regenerate'})),
     # admin sharing
-    url(r'^admin/perms/(?P<username>[-_\w]+)/?$',
+    url(r'^admin/perms/(?P<username>.+)/?$',
```

Comment:
Since the underlying Django model has restrictions on allowable characters for `username`, we should probably match that rather than allow all chars. [That regex](https://github.com/django/django/blob/master/django/contrib/auth/validators.py#L21) looks like `r'^[\w.@+-]+$'`.